metalinter.json

```diff
@@ -0,0 +1,13 @@
+{
+  "Linters": {
+	      "errcheck": "errcheck -abspath -ignore 'Close|Write|Serve' {path}:PATH:LINE:COL:MESSAGE",
```

Comment:
errcheck currently ignores `'Close|Write|Serve,os:Remove'`. Not sure if `os:Remove` is used.